Utils/Elasticsearch/requests/event_summary.json

```diff
@@ -0,0 +1,49 @@
+/*
+  Event Summary Report query:
+  Query search by:
+    - campaign
+    - task status
+    - hashtag
+  Outer aggregation: by phys_category
+  Inner aggregation: by step_name (production step)
+  Aggregated parameters:
+    - requested_events (sum)
+    - processed_events (sum)
+*/
+GET /prodsys_rucio_ami/task/_search
+{
+  "size": 0,
+   "query": {
+     "bool": {
+       "must": [
+         { "term": { "campaign.keyword": "<CAMPAIGN>" } },
+         { "term": { "status": "<STATUS>" } }
+       ],
+       "should": [
+         { "term": { "hashtag_list": "<HASHTAG>"} }
+       ]
```

Comment:
ES documentation [says](https://www.elastic.co/guide/en/elasticsearch/reference/5.4/query-dsl-bool-query.html):
>If the `bool` query is in a query context and has a `must` or `filter` clause then a document will match the bool query even if none of the `should` queries match. In this case these clauses are only used to influence the score.

In other words, `should` is used to put documents with given <HASHTAG> on top of the search results -- but in the summary it is as good as nothing. If we want to search by hashtag, it should also be presented in `must` clause, if not -- removed.